service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/CaseSubmissionDocumentAnnexGenerator.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.hmcts.reform.fpl.service.docmosis;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.enums.ApplicationDocumentType;
+import uk.gov.hmcts.reform.fpl.model.ApplicationDocument;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.model.docmosis.DocmosisAnnexDocument;
+import uk.gov.hmcts.reform.fpl.model.docmosis.DocmosisAnnexDocuments;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.nullSafeList;
+
+@Component
+public class CaseSubmissionDocumentAnnexGenerator {
+
+    public DocmosisAnnexDocuments generate(CaseData caseData) {
+
+        List<Element<ApplicationDocument>> documents = caseData.getApplicationDocuments();
+
+        return DocmosisAnnexDocuments.builder()
+            .featureToggleOn(true)
```

Comment:
true or dependant on the feature toggle service?

Reply:
the feature toggle is external, see 
 .annexDocuments(
                featureToggleService.isApplicationDocumentsEventEnabled()
                    ? annexGenerator.generate(caseData) : buildDocmosisAnnexDocuments(caseData)
            ).

Reply:
my bad, am blind. Monday morning things 🙂